cl/api/templates/webhooks-docs-vlatest.html

```diff
@@ -293,6 +294,65 @@ <h3 id="search-alerts">Search Alert Events</h3>
   --url '{% get_full_host %}{% url "alert-list" version="v3" %}' \
   --header 'Authorization: Token {% if user.is_authenticated %}{{ user.auth_token }}{% else %}&lt;your-token-here&gt;{% endif %}' </pre>
 
+  <h3 id="old-da-alerts-report">Old Docket Alerts Report Events</h3>
+  <p>We sent a weekly email report for <a href="{% url 'alert_help' %}#recap-alerts">Docket Alerts</a> on long-stagnant cases, this report might include:
+  </p>
+  <ul>
+    <li>Automatically Disabled Alerts</li>
+    <li>Old Terminated Cases and Cases</li>
+    <li>Terminated about 90 Days Ago</li>
+  </ul>
+  <p>If you have an Old Docket Alerts Report webhook <a href="{% url 'webhooks_getting_started' %}#setup-a-webhook">endpoint enabled</a>, you’ll receive a webhook event with the same weekly report you received by email.
+  </p>
+  <p>The webhook event is a JSON hash with two keys, <code>webhook</code> and <code>payload</code>. <code>payload</code> has three keys:</p>
+  <ul>
+    <li>
+      <p><code>disabled_alerts</code> (Automatically Disabled Alerts):</p>
+      <p>Contains a list of docket alerts that have been disabled automatically by our system on terminated cases that haven’t had updates for over 180 days. You can re-enable these alerts if they were disabled by mistake.</p>
+    </li>
+    <li>
+      <p><code>very_old_alerts</code> (Old Terminated Cases):</p>
+      <p>Contains a list of docket alerts terminated cases that were last updated about 180 days ago. These alerts will be disabled in a week if you don’t take action. If you are still monitoring these cases please disable and re-enable these alerts. Doing so will keep them running for another 180 days.</p>
+    </li>
+    <li>
+      <p><code>old_alerts</code> (Terminated about 90 Days Ago):</p>
+      <p>Contains a list of docket alerts on terminated cases that were last updated about 90 days ago. If you are no longer monitoring these cases, you may want to disable these alerts.</p>
+    </li>
+  </ul>
```

Comment:
```suggestion
  <h3 id="old-da-alerts-report">Old Docket Alert Events</h3>
  <p>If a case stops receiving updates due to being terminated or otherwise dormant, we automatically disable any <a href="{% url 'alert_help' %}#recap-alerts">docket alerts</a> that may be configured for it. This helps ease the load on our servers, and helps users in our free tier identify cases that they may wish to stop following.
  </p>
  <p>To help our users manage this, we send a weekly email telling our users about any docket alerts they have that will soon be automatically disabled. This allows them to take one of three actions on the alert:
  </p>
  <ol>
    <li>
      <p><strong>Do nothing</strong> &mdash; If the user does this, the alert will soon be deleted during the next week's automatic process.
      </p>
    </li>
    <li>
      <p><strong>Delete the alert</strong> &mdash; The user will immediately stop getting alerts for the case.</p>
    </li>
    <li>
      <p><strong>Re-up the alert</strong> &mdash; This tells us that the user wishes to continue getting alerts for the case and pushes out the automatic disablement by about six months.</p>
    </li>
  </ol>
  <p>By subscribing to this webhook your server will get a notification similar to the weekly email we send to users. At that time, you can decide whether to continue monitoring stagnant cases (by re-upping them) or let the automatic disablement occur.
  </p>
  <p>As with our other events, this webhook event is a JSON hash with two keys, <code>webhook</code> and <code>payload</code>. <code>payload</code> has two keys:</p>
  <ul>
    <li>
      <p><code>disabled_alerts</code> (Automatically Disabled Alerts):</p>
      <p>Contains a list of docket alerts that have been automatically disabled by our system on terminated cases that haven’t had updates for over 180 days. You can re-enable these alerts if they were disabled by mistake.
      </p>
    </li>
    <li>
      <p><code>old_alerts</code> (Old Terminated Cases):</p>
      <p>Contains a list of docket alerts on terminated cases that were last updated about 180 days ago. These alerts will be disabled during the next week's process if you do not re-up them for another six months.
      </p>
    </li>
  </ul>
```

Reply:
Changed, thanks!